ui/src/interfaces/widget.ts

```diff
@@ -1,3 +1,7 @@
 export enum CustomFieldTypes {
     MultipleSelect = 'multipleSelect'
-}
\ No newline at end of file
+}
+
+export enum CustomWidgetTypes {
+    Header = 'Header'
```

Comment:
Why we not reusing built-in [HeaderWidget](https://github.com/tesler-platform/tesler-schema/blob/b7b6a9b4874d82cd2a010f6370b275fe33375121/src/interfaces/widget.ts#L28)?

Reply:
@Dergash HeaderWidget is used